templates/zerver/help/zulip-glossary.md

```diff
@@ -49,6 +49,13 @@ phrase (a complete list of emojis can be found
 the **smiley face** (<i class="icon-vector-smile"></i>) icon below the
 [compose box](#compose-box).
 
+### emoji reaction
+
+**Emoji reactions** are small [emoji](#emoji) icons that appear below messages.
+Although they are generally used to show reactions to a message, they can be
+used for various other purposes, such as voting or checking off items in a
```

Comment:
The `checking off items in a to-do list` is a great usage example, do we have it documented anywhere? I'm not sure how you'd do that :)